components/error-card.tsx

```diff
@@ -0,0 +1,27 @@
+import { Card, Group, Text, Badge, Container } from "@mantine/core";
+
+/**
+ * TODO Please improve the error component / make the border red or something
```

Comment:
@JohannesMerkt  I was trying to style this card but I couldn't figure it out ...  I wanted it to look more like error, maybe red border?